File: 2_java/Main_2525.java
import java.util.Scanner;

public class Main_2525 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int hour = input.nextInt();
        int min = input.nextInt();
        int work = input.nextInt();

        int totalMin = hour*60 + min + work;

        if (totalMin<1440) {
            System.out.println((totalMin/60)+" "+(totalMin%60));
        }
        else {
            System.out.println(((totalMin-1440)/60)+" "+(totalMin%60));
        }

        input.close();



    }
}

File: 2_python/Main_2525.py
hour,min = map(int,input().split())
work_min = int(input())

total_min = hour*60 + min + work_min

if total_min<1440 :
    print((total_min//60),(total_min%60))

else :
    print(((total_min-1440)//60),(total_min%60))